tests/behat/src/AdminContext.php

```diff
@@ -22,8 +22,8 @@ public function iShouldSeeAnInvalidTabIcon(string $not, string $tabLabel)
     {
         $id = str_replace('Main Content', 'Main', $tabLabel ?? '');
         $id = str_replace(' ', '_', $id ?? '');
-        $id = "tab-Root_$id";
-        $selector = "[aria-labelledby=$id] .font-icon-attention-1";
+        $id = "Root_$id";
+        $selector = "[aria-labelledby=tab-$id] .font-icon-attention-1, li[aria-controls=$id] .font-icon-attention-1";
```

Comment:
This is needed so we can check the "invalid tab icon" in a `GridField` edit form